src/libtelio.udl

```diff
@@ -556,8 +556,8 @@ dictionary Features {
 };
 
 dictionary FeatureBatching {
-    /// direct connection threshold for batching
-    u32 direct_connection_threshold;
+    /// Connection threshold for batching
+    u32 threshold;
```

Comment:
Adding reference to this: https://github.com/NordSecurity/libtelio/pull/772/files#r1751581554